test/jit/test_freezing.py

```diff
@@ -1939,7 +1950,42 @@ def test_layernorm(self):
 
                 for layernorm in (torch.nn.LayerNorm(param[1]),
                                   torch.nn.LayerNorm(param[1], elementwise_affine=False)):
-                    sub_model = torch.nn.Sequential(param[0], layernorm)
+                    # to generate non inplace tests we extend the use of layernorm's input
+                    for inplace in (True, False):
+                        sub_model = torch.nn.Sequential(param[0], layernorm) if inplace else ResidualLayernorm(param[0], layernorm)
+                        sub_model.eval()
+                        mod = torch.jit.freeze(torch.jit.script(sub_model))
+                        self.run_pass("convert_frozen_ops_to_mkldnn", mod.graph)
+                        # if weight and bias are present and shape is the last dimension
+                        # we should convert `aten::layer_norm` to `prim::MKLDNNLayerNorm`
+                        if layernorm.elementwise_affine and len(param[1]) == 1:
+                            inplace_suffix = "_" if inplace else ""
+                            FileCheck().check("prim::MKLDNNLayerNorm" + inplace_suffix).check_count("aten::to_dense", 1, exactly=True).run(mod.graph)
+                        else:
+                            FileCheck().check_count("aten::to_dense", 1, exactly=True).run(mod.graph)
+                        self.assertTrue(torch.allclose(sub_model(param[2]), mod(param[2]), 1e-04, 1e-04))
+
+    @skipIfNoTorchVision
+    def test_layernorm2(self):
```

Comment:
can you have a more descriptive testname ? what is this testing that the other test is not